### ISSUE TITLE
Uninitialized variable in HVACWaterToAirMultiSpeedHP (CR #9322)

### ISSUE THREAD

**Issue by axelstudios** [NREL/EnergyPlus#4093]
###### Added on 2013-09-28 10:00 by @lklawrie
--
#### Description
The c++ conversion guy pointed out:

HVACWatertoAirMultiSpeedHP.f90:SizeVarSpeedCoil
 SensCapTempModFac is used uninitialized (only in error messages)

This is occuring only in error messages 

[30 Sept 2013  BAN ]
The variable "SensCapTempModFac" defined in Multispeed HP module but never initialized and never used in the calculation; however, it was included in the warning error message.  This part of the error message is not required for the variable speed/multispeed water to air DX cooling coils. It looks like the error message was adopted from single speed water to air HP module inadvertently. So, I commented out the variable and modified the warning error message.

Also found additional problem during this investigation.
(1) When the total cooling capacity is hardsized, the sensible cooling capacity is reported as the "autosize" value, which is -99999.0 and results in negative sensible heat ratio and latent capacity greater than the hardsized total cooling capacity.  I have found this problem in the following six test suite files.

HeatPumpVSAS.idf
PackagedTerminalAirConditionerVSAS.idf
PackagedTerminalHeatPumpVSAS.idf
UnitarySystem_VSHeatPumpWaterToAirEquationFit.idf
VSHeatPumpWaterToAirEquationFit.idf
VSHeatPumpWaterToAirWithRHControl.idf

(2) Also the predefined table entry routine is called in getinput and sizing. This is redundant. Since the sensible Cooling capacity of Multispeed and variable speed DX cooling coils is always autosized, now the predefined table entry is called for each coil and it is done in sizing routine only.
  
Test file: any of the six test suite files listed above.
Weather file: any

LKL, 8 Oct 2013
Mike -- I'm sorry, it looks like Bereket went the extra mile here but I just don't have time to sort it all out.

--
Addressed in `13.10.03 V8.1.0.005` by @Nigusse
Last build tested: `13.08.23 V8.1.0.004`

**Comment by Nigusse** [comment]
@Myoldmopar @mjwitte.  I created a branch for this bug and tested six of the IDFs that I identified for this bug more than a year ago. It happened that I also have the old result files (CR9322) to compare to. I remembered that I addressed the uninitialized variable issue then.  Issues # 1 and 2 must have been addressed by someone else that I could not duplicate them. This issue does not warrant any further investigation and shall be closed.  

@Myoldmopar Do I need to create a story for this bug on pivotal? Should I simply close the bug issue #4093 in github?
